src/waveform/renderers/waveformmark.h

```diff
@@ -47,10 +38,13 @@ class WaveformMark {
     int getHotCue() const { return m_iHotCue; };
     void setHotCue(int hotCue) { m_iHotCue = hotCue; };
 
+    virtual ~WaveformMark(); //set the destructor which will use the reset functionallity and do the same thing
```

Comment:
We have no other virtual functions. So the destructor does not need to be virtual.